db/migrate/20230606140456_create_like.rb

```diff
@@ -0,0 +1,11 @@
+class CreateLike < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+      t.string :timestamp
+      t.references :users, null: false, foreign_key: true, index: true
```

Comment:
To follow the ERD correctly, the foreign key is supposed to be an AuthorId. So recommend changing the user to author like this:

```
`t.references :author, foreign_key: { to_table: 'users' }`
```